tcf_website/static/base/bugform.js

```diff
@@ -0,0 +1,31 @@
+function submitForm(){
+  var form = document.getElementById("bugform");
+  var valid = validateForm(form);
+  if (valid === true) {
+    postToDiscord();
+    // reset form after submit
+    $('#bugModal').modal('toggle');
+    form.classList.remove('was-validated');
+    var descriptionField = document.getElementById("descriptionField");
+    descriptionField.value = "";
+  }
+}
+
+function postToDiscord(){
+  var url = $("#urlField").val();
+  var email = $("#emailField").val();
+  var description = $("#descriptionField").val();
+  var data = {
+    "content": "Bug Found! \n**URL:** " + url + "\n**Description**: \n" + description + "\n**Email:** " + email
+  }
+
+  var discordURL = "https://discordapp.com/api/webhooks/767189878223142942/wwBQA0K4VQ0i94ku2os2tYIVpPbDtfeP1i6s5G3CBWSCI7R0t6PbhZxwgJ8Z2yYpyv-q"
```

Comment:
Pretty sure we don't want this URL to be public? Maybe even regenerate a key (if that's possible? not sure how it works) if you can and then save it in `.env` (add it to the tCF secrets doc as well); it's highly unlikely that anybody would find it anyways, but I think it's good practice.

Reply:
yeah probably not, for the .env to work here we would need to `npm install dotenv`, so I was trying to avoid that initially, should be a quick fix though

Reply:
@j-alicia-long Looks like another push towards including npm in the repo...

Reply:
Turns out you can only access the `.env` on the backend so no npm needed. Time to learn Django REST tho

Reply:
> @vn6 Can you add `DISCORD_URL_BUG` to the .env in [the tCF drive](https://docs.google.com/document/d/1HsuJOf-5oZljQK_k02CQhFbqw1q-pD_1-mExvyC1TV0/edit)?

will do after I get it working :)

Reply:
done